core/core_modules/kpm.js

```diff
@@ -287,29 +288,30 @@ var git = require.once('../git.js'),
 
                 for (var i = 0; i < args.length; i++) {
                     var url = args[i];
+                    var spl = url.split('/');
+                    if (spl.length === 1) {
+                        refreshModuleTable(url, function(u, err) {
+                                if (err || !moduleTable.modules[u]) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 28 space characters but found 32.](https://www.codacy.com/app/theprojectshop/Kassy/file/2719763881/issues/source?bid=3282987&fileBranchId=3347522#l294)